File: java/neetcode_all/01_arrays_&_hashing/0027_remove_element.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int k = 0;
        for(int i=0; i < nums.length; i++) {
            if (nums[i] != val) {
                nums[k] = nums[i];
                k += 1;
            } 
        }
        return k;    
    }
}

File: javascript/0027-remove-element.js
var removeElement = function(nums, val) {
 
    let i = 0;
     
    do {
      if(nums[i] === val)  {
          nums.splice(i,1);
       
      } else {
          i++;
      }
        
    }while(i < nums.length)
        
        return nums.length;
};
